src/App.tsx

```diff
@@ -1,25 +1,109 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+
+export const App: React.FC = () => {
+  const getUser = (param: number | string): User | null => {
+    const foundUser
+    = typeof param === 'number'
+      ? usersFromServer.find(user => user.id === param)
+      : usersFromServer.find(user => user.name === param);
+
+    return foundUser || null;
```

Comment:
```suggestion
    const foundUser = usersFromServer.find(user => (
       typeof param === 'number'
         ? user.id === param
         : user.name === param
    ))
    
    return foundUser || null;
```

Reply:
but anyway it does not seem to be logical - looking for the user by their name. Better to stick to id, cause anyway every user has it